__tests__/src/packages/imperative/plugins/suites/ValidatePlugin.ts

```diff
@@ -221,6 +345,24 @@ describe("Validate plugin", () => {
                 expect(result.stdout).toContain("The plugin's configuration does not contain an 'imperative.rootCommandDescription' property.");
             });
 
+            it("missing rootCommandDescription property - warning", () => {
```

Comment:
This test looks like it may be for an error rather than a warning.

Reply:
Yep, you're right. Good catch.